app/models/incomeSourceDetails/UIJourneySessionData.scala

```diff
@@ -95,11 +95,17 @@ object ManageIncomeSourceData {
 }
 
 case class CeaseIncomeSourceData(
-                                  selectedIncomeSourceId: String,
-                                  dataCeased: Option[LocalDate]
+                                  incomeSourceId: Option[String],
+                                  endDate: Option[String],
```

Comment:
* what is the reason to change dateCeased from type LocalDate to String ? 
* Type String is not correct representation of the date; if you need any logic to correctly convert LocalDate to any string format, use existing helper method or any a new one;

Reply:
All the code at the moment uses Strings so I thought replacing everything with LocalDates would be outside the scope of the ticket

Reply:
I am not sure what you mean by "all the code", there should not be a problem to deal with LocalDate